Weather/Scenes/Weather/WeekForecastCollectionViewCell.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  WeekForecastCellView.swift
+//  Weather
+//
+//  Created by Vladislav Kondrashkov on 10/22/18.
+//  Copyright © 2018 Vladislav Kondrashkov. All rights reserved.
+//
+
+import UIKit
+
+final class WeekForecastCollectionViewCell: UICollectionViewCell {
+    private var weekDayNameLabel = UILabel()
+    private var temperatureAtMiddayLabel = UILabel()
+    private var temperatureAtNightLabel = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        weekDayNameLabel.font = .boldSystemFont(ofSize: 17)
+        weekDayNameLabel.textColor = .white
+        contentView.addSubview(weekDayNameLabel)
+        activateWeekdayLabelConstraints(view: weekDayNameLabel)
+        
+        temperatureAtNightLabel.font = .boldSystemFont(ofSize: 17)
+        temperatureAtNightLabel.textColor = .white
+        contentView.addSubview(temperatureAtNightLabel)
+        activateTemperatureAtNightLabelConstraints(view: temperatureAtNightLabel)
+        
+        temperatureAtMiddayLabel.font = .boldSystemFont(ofSize: 17)
+        temperatureAtMiddayLabel.textColor = .white
+        contentView.addSubview(temperatureAtMiddayLabel)
+        activateTemperatureAtMiddayLabelConstraints(view: temperatureAtMiddayLabel, anchorView: temperatureAtNightLabel)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func updateWeekDayNameLabel(text: String) {
+        weekDayNameLabel.text = text
+    }
+    func updateTemperatureAtMiddayLabel(text: Int) {
+        temperatureAtMiddayLabel.text = String(text)
+    }
+    func updateTemperatureAtNightLabel(text: Int) {
+        temperatureAtNightLabel.text = String(text)
+    }
+    
+    func display(weekDay: String, temperatureAtMidday: Int, temperatureAtNight: Int) {
```

Comment:
Move the subview's setup to the init and leave it as only setters to the subviews. Also remove other `update` methods